src/core/edgeNetwork/injectProcessWarningsAndErrors.js

```diff
@@ -10,21 +10,37 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const MESSAGE_PREFIX = "The server responded with the following";
+import { NO_CONTENT } from "../../constants/httpStatusCode";
+
+const MESSAGE_PREFIX = "The server responded with a";
```

Comment:
I decided to use this message prefix for fatal errors, non-fatal errors, and warnings. This resulted in some of the text changes you see in tests.